src/components/ContainerSettingsVolumes.react.js

```diff
@@ -15,12 +15,21 @@ var ContainerSettingsVolumes = React.createClass({
       }
 
       var directory = filenames[0];
-
-      if (!directory || directory.indexOf(util.home()) === -1) {
+      var founded = false;
+        if(directory){
```

Comment:
Can you fix the indentation and check that spacing around keywords 'if' are respected?